src/prompt.js

```diff
@@ -209,6 +209,17 @@ export async function promptForHerokuCredentials() {
 
       return true;
     }
+  }, {
+    name: 'apiKey',
+    type: 'password',
+    message: 'Heroku API Key:',
+    validate: (value) => {
+      if (typeof value === 'undefined' || value === '' || value.indexOf(' ') !== -1) {
```

Comment:
You should trim whitespace from `value` before checking for spaces.

Reply:
Of course, this would mean the value has to also be trimmed elsewhere when it gets used...